Channels/Integration/01d_Amazon.md

```diff
@@ -0,0 +1,220 @@
+
+9. Select the checkbox with the notice *I request Amazon to grant Actindo access to my seller account and the corresponding data. I am responsible for all measures that may be taken by this application.*, and click  the [Confirm] button.  
+
+    A new window with the notice *We authorize Actindo to access your seller details.* is displayed. You are redirected back to the *Actindo Core1 Platform*. The *Access token* field has been automatically filled in.
+
+    ![Access token](../../Assets/Screenshots/Channels/Settings/Connections/Amazon/AccessToken.png "[Access token]")
+
+[comment]: <> (Selber übersetzt. Nicht sicher, was eigentlich steht. Nach Screenshots fragen, wenn möglich.)
+
+10. Click the *Amazon marketplace* drop-down list and select the applicable marketplace.
+    > [Info] Once it has been saved, the selected marketplace cannot be edited anymore.
+
+11. Click the [SAVE] button.  
+    The *Checking credentials* notice is displayed. The connection is established and the synchronization is triggered. The *Sync triggered* pop-up window is displayed. 
+
+    ![Sync triggered](../../Assets/Screenshots/Channels/Settings/Connections/SyncTriggered.png "[Sync triggered]")
+
+    The *CONNECTIONS* tab in the *Settings* menu entry of the *Omni-Channel* module is displayed when the connection has been established. The *Amazon* connection is displayed in the list of connections.
+
+    ![Amazon connection](../../Assets/Screenshots/Channels/Settings/Connections/Amazon/Connection.png "[Amazon connection]")
+
+[comment]: <> (Letzter Satz von Shopify genommen. Evtl. mit Result oben -11- integrieren?)
+
+12. If necessary, continue to [Configure the Amazon connection](#configure-the-amazon-connection).
+
+
+## Configure the Amazon connection   
+
+After the connection to an Amazon shop has been established, further settings can be configured for the connection.
+
+#### Prerequisites
+
+An Amazon connection has been established, see [Create an Amazon connection](#create-an-amazon-connection).
+
+#### Procedure
+
+*Omni-Channel > Settings > Tab CONNECTIONS*
+
+![Amazon connection](../../Assets/Screenshots/Channels/Settings/Connections/Amazon/Connection.png "[Amazon connection]")
+
+1. Click the Amazon connection in the list of connections.   
+  The *Edit connection* view is displayed. By default, the *Credentials* tab is displayed.
+
+    ![Edit connection credentials](../../Assets/Screenshots/Channels/Settings/Connections/Amazon/Credentials.png "[Edit connection credentials]")
+
+2. Click the *Settings* tab.   
+  The *Settings* tab is displayed. By default, the *Offers* setting is displayed. In the right side bar, the *Apply from default* toggle is preselected and the *Update stock for 'Fulfillment by Amazon (FBA)' offers* is locked.  
+
+    > [Info] It is recommended to leave this settings unchanged, since Amazon manages stock for FBA offers.
```

Comment:
```suggestion
3. If necessary, disable the *Apply from default* toggle.  By default, the toggle is enabled.
    The *Update stock for 'Fulfillment by Amazon (FBA)' offers* toggle is unlocked.
       
 4. Enable the *Update stock for 'Fulfillment by Amazon (FBA)' offers* toggle to manage the stock also via Actindo. It is recommended to leave this toggle disabled, since Amazon manages the stock for FBA offers.
```